src/app.ts

```diff
@@ -1,20 +1,32 @@
 import * as express from "express";
 import * as logger from "morgan";
 import * as path from "path";
+import * as cors from "cors";
 
 import { errorHandler, errorNotFoundHandler } from "./middlewares/errorHandler";
+import { attachCORSResponse } from './middlewares/proxyCORS';
 
 // Routes
 import { index } from "./routes/index";
 // Create Express server
 export const app = express();
 
+/**
+ * @TODO Add more details such such as whitelists, methods options, etc
+ */
+app.use(cors());
+
 // Express configuration
 app.set("port", process.env.PORT || 3030);
 
 app.use(logger("dev"));
 
-app.use(express.static(path.join(__dirname, "../public")));
+// No Template Engine is Defined
+// app.use(express.static(path.join(__dirname, "../public")));
```

Comment:
Is there a reason to disable this? I figured that serving `/robots.txt` (which should currently be in the `public` folder) might be useful for the API to prevent crawling.

Reply:
Not really, we just haven't downloaded an template engine modules like pug or ejs and it was throwing me an error everytime I tried to run it. I believe we just need to download one of them and specify it as a middleware before we having express server static files or include an index.html file in the public folder so that it can default onto something.